tycho/db/event/__init__.py

```diff
@@ -1,14 +1,20 @@
+import asyncio
 import pymongo
+import queue as q
+
+from pymongo.errors import DuplicateKeyError
+from aiohttp.web import HTTPNotFound
+from transmute_core import APIException
+
 from ..utils import async_generator
 from ...models.eventnode import EventNode
 from ...models.event import Event as ModelEvent
-from aiohttp.web import HTTPNotFound
-import queue as q
-
 from .deserialize import deserialize_db_event
 from .serialize import serialize_to_db_event
 
+
 MAX_RECURSIVE_DEPTH = 100
+WAIT_TIME_S = 1
```

Comment:
can we expand this variable name? there's not much valuable in abbreviating it to "s". It's not the most ambiguous, but "WAIT_TIME_SECONDS" is definitely clearer.

Reply:
Updated the variable name to `WAIT_TIME_SECONDS`